lib/fs-loader.js

```diff
@@ -143,6 +143,121 @@ const load = async (dir, options, config) => {
+//      processors <Object>
+//        [type] <Object>
+//          add <Function>
+//            schema <Schema>
+//            ms <Metaschema>
+//            Returns: <Error[]>
+//          postprocess <Function>, optional
+//            schema <Schema>
+//            ms <Metaschema>
+//            Returns: <Error[]>
+//          validateInstance <Function>, optional
+//            ms <Metaschema>
+//            schema <Schema>
+//            instance <any>
+//            options <Object>
+//            Returns: <Error[]>
+//      processOrder: <Function> | <Object>, function is passed to
+//          Array.prototype.sort (a: <Schema>, b: <Schema>) => <number>.
+//          If <Object> is provided it would be used as
+//          map from schema type (<string>) to order value (<number>),
+//          types with lower values are processed earlier.
+//   params <Object>
+//     options <Object>
+//       decorators
+//       localDecorators <Object>, optional
+//         [type] <Object> decorators specific for this schema type
+//       pathToType <Object>
+//         [ext] <string>
+//       pathToType <Function>
+//         path <string>
+//       Returns: <string>
+//       loadOrder <Function>
+//         a <string>
+//         b <string>
+//         Returns: <number>
+//     config <Object>
+//      prepare <Function>
+//        ms <Metaschema>
+//      resolve <Function>
+//        ms <Metaschema>
+//        path <string>
+//        Returns: <Schema>
+//      processors <Object>
+//        [type] <Object>
+//          add <Function>
+//            schema <Schema>
+//            ms <Metaschema>
+//            Returns: <Error[]>
+//          postprocess <Function>, optional
+//            schema <Schema>
+//            ms <Metaschema>
+//            Returns: <Error[]>
+//          validateInstance <Function>, optional
+//            ms <Metaschema>
+//            schema <Schema>
+//            instance <any>
+//            options <Object>
+//            Returns: <Error[]>
+//      processOrder: <Function> | <Object>, function is passed to
+//          Array.prototype.sort (a: <Schema>, b: <Schema>) => <number>.
+//          If <Object> is provided it would be used as
+//          map from schema type (<string>) to order value (<number>),
+//          types with lower values are processed earlier.
+// Returns: workParam <Object>
+const readConfig = async (path, systemParams, params) => {
+  const workParam = common.clone(params);
+  const systemSchema = await load(
+    path,
+    systemParams.options,
+    systemParams.config
+  );
+  for (const type of Object.keys(workParam.config.processOrder)) {
+    if (systemSchema.categories.has(type)) {
+      workParam.config.processors[type].validateSchema = [
+        schema => {
+          const error = systemSchema.validate(
+            'category',
+            type,
+            schema.definition,
+            {
+              path: `${schema.name}.`,
+            }
```

Comment:
I think we should extract this `options` object to a variable to be able to write this call in one line. This should make it a bit clearer.